static-catalog/implementations/DigitalGlobe/examples/DigitalGlobe_1B_product_mul.json

```diff
@@ -0,0 +1,198 @@
+{
+  "id": "103001004B316600_P002_MUL",
+  "type": "Feature",
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
+      [
+        [
+          [
+            -105.160846396,
+            39.1656009422
+          ],
+          [
+            -104.947574355,
+            39.1750024943
+          ],
+          [
+            -104.949890373,
+            39.0512153242
+          ],
+          [
+            -105.161151214,
+            39.0436821016
+          ],
+          [
+            -105.160846396,
+            39.1656009422
+          ]
+        ]
+      ]
+    ]
+  },
+  "assets": [
+    {
+      "uri": "https://15NOV09180446-M1BS-056823192010_01_P002.TIF",
```

Comment:
Would these link to a real location? Or are they just supposed to unique references? Like in a production version would this be a real url that a user (with appropriate rights) would follow?

Reply:
Yeah, they would link to actual URLs.  I just truncated this example.